include/psa/crypto_extra.h

```diff
@@ -48,6 +48,7 @@ extern "C" {
  * @{
  */
 
```

Comment:
Missing: removing the `alg2` field from the context, and the code in `psa_crypto.c` that deals with it.

Reply:
True, that's something I noticed as well. I didn't modify that part because, as I explained in my [comment below](https://github.com/Mbed-TLS/mbedtls/pull/8449#issuecomment-1790233521), I was mostly focused on having a TLS/DTLS implementation which could work with standard PSA interface.
However I agree that this is something I can improve as well. I will try to fix this ASAP